src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {person.name}
+      </h2>
+      <p className="Person__age">{person.age && (`I am ${person.age}`)}</p>
```

Comment:
Omit age if it is not given. Current implementation will render an empty paragraph if age is not given.